### PR TITLE
Updating Jaeger code blocks to OpenShift standards.

### PR DESCRIPTION
**Pull request by JStickler**
https://issues.redhat.com/browse/OSSMDOC-114

Separate Code Block Command from Output

Change indented codeblocks to fenced

### CODE REVIEW

File: modules/jaeger-removing-instance-cli.adoc [hunk 1]
```diff
@@ -9,20 +9,29 @@ This PROCEDURE module included in the following assemblies:
...
+[source,terminal]
 ----
 $ oc login
 ----
 +
 . To display the Jaeger instances run the command:
 +
+[source,terminal]
 ----
 oc get deployments -n <jaeger-project>
```
Comment (neal-timpe) [inline modules/jaeger-removing-instance-cli.adoc:21]: Does this need a `$`?

File: modules/jaeger-removing-instance-cli.adoc [hunk 2]
```diff
@@ -9,20 +9,29 @@ This PROCEDURE module included in the following assemblies:
...
 ----
 +
 The names of operators have the suffix `-operator`. The following example shows two Jaeger Operators and four Jaeger instances:
 +
+[source,terminal]
 ----
 oc get deployments -n jaeger-system
```
Comment (neal-timpe) [inline modules/jaeger-removing-instance-cli.adoc:28]: Does this need a `$`?

File: modules/jaeger-removing-instance-cli.adoc [hunk 3]
```diff
@@ -34,28 +43,38 @@ tracing2                 1/1     1            1           35m
 +
 . To remove an instance of Jaeger, run the command:
 +
+[source,terminal]
 ----
-oc delete jaeger <deployment-name>  -n <jaeger-project>
+oc delete jaeger <deployment-name> -n <jaeger-project>
```
Comment (neal-timpe) [inline modules/jaeger-removing-instance-cli.adoc:48]: Does this need a `$`?

File: modules/jaeger-removing-instance-cli.adoc [hunk 4]
```diff
@@ -34,28 +43,38 @@ tracing2                 1/1     1            1           35m
...
 ----
 +
 For example,
 +
+[source,terminal]
 ----
 oc delete jaeger tracing2 -n jaeger-system
```
Comment (neal-timpe) [inline modules/jaeger-removing-instance-cli.adoc:55]: Does this need a `$`?

File: modules/jaeger-removing-instance-cli.adoc [hunk 5]
```diff
@@ -34,28 +43,38 @@ tracing2                 1/1     1            1           35m
...
 ----
 +
 
-. To verify the deletion, run `oc get deployment` again :
+. To verify the deletion, run `oc get deployment` again:
 +
+[source,terminal]
 ----
 oc get deployments -n <jaeger-project>
```
Comment (neal-timpe) [inline modules/jaeger-removing-instance-cli.adoc:63]: Does this need a `$`?

File: modules/jaeger-removing-instance-cli.adoc [hunk 6]
```diff
@@ -34,28 +43,38 @@ tracing2                 1/1     1            1           35m
...
 ----
 
 +
 For example,
 +
+[source,terminal]
 ----
 oc get deployments -n jaeger-system
```
Comment (neal-timpe) [inline modules/jaeger-removing-instance-cli.adoc:71]: Does this need a `$`?

Comment (JStickler) [reply modules/jaeger-removing-instance-cli.adoc]: OMG, I must have been completely out of $$$ that day.  Fixed.

**Review by neal-timpe — APPROVED** [review]

**Pull request merged**